pyproject.toml

```diff
@@ -441,6 +436,7 @@ runtime-evaluated-base-classes = ["tmt.config.models.BaseConfig"]
 
 [tool.ruff.lint.isort]
 known-first-party = ["tmt"]
+split-on-trailing-comma = false
```

Comment:
I think we want `true` here to convince `ruff` that if there is a trailing command it should not wrap import lines into a single one. With `false` it seems to collapse the list into a single line.

Reply:
I think I've set it in order to minimise changes, but yeah it should be dropped

Reply:
Ack, fixed in 8dc911bd.